core/templates/services/exploration-features.service.ts

```diff
@@ -17,25 +17,35 @@
  *               the exploration editor.
  */
 
-import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+export interface IExplorationDataDict {
+  'param_changes': string[];
```

Comment:
You don't need single quotes in the properties of the interfaces.

Reply:
removed the single quotes
PTAL